test/test-filter-process.sh

```diff
@@ -6,15 +6,7 @@
 # from the "next" branch, which is the only (current) version of Git that has
 # support for the filter protocol.
 #
-# Once 2.11 is released, replace this with:
-#
-# ```
-# ensure_git_version_isnt $VERSION_LOWER "2.11.0"
-# ```
-if [ "1" -ne "$(git version | cut -d ' ' -f 3 | grep -c "g")" ]; then
-  echo "skip: $0 git version does not include support for filter protocol"
-  exit
-fi
+ensure_git_version_isnt $VERSION_LOWER "2.11.0"
```

Comment:
This seems to work but it still generates a few error messages:
```
test/testhelpers.sh: line 416: 10#rc2: value too great for base (error token is "10#rc2")
test/testhelpers.sh: line 420: 10#rc2: value too great for base (error token is "10#rc2")
```
See: https://travis-ci.org/git-lfs/git-lfs/jobs/177907753

... Oh, a few lines down I read this:
```
skip: test/test-install-custom-hooks-path-unsupported.sh (git version > 2.9.0)
```
The git version is clear greater than 2.9, right?

However, this is a separate issue and unrelated to the 1024 error.